presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixDistributedQueries.java

```diff
@@ -136,8 +137,13 @@ public void testCreateSchema()
     @Override
     public void testColumnName(String columnName)
     {
-        // TODO (https://github.com/prestosql/presto/issues/3466) Phoenix generally lacks quoting in underlying queries
-        throw new SkipException("TODO");
+        if (columnName.equals("a\"quote")) {
+            assertThatThrownBy(() -> super.testColumnName(columnName))
+                    .hasMessageContaining("Illegal data. Unsupported sql type: QUOTE");
```

Comment:
That's incorrect error message. We didn't ask to create a column of _type_ `quote`.

This is because `getEscapedArgument` doesn't support escaping `"` within the name.
If Phoenix does not support this, we need to filter this out on our side.

Then, the correct way to exclude this test case would be like here
https://github.com/prestosql/presto/blob/329a48592ecb6c628712ef3648f2e8da7eb33d38/presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java#L123-L127

Reply:
@findepi Replaced with `isColumnNameRejected`. As far as I confirmed, we cannot use double-quotation for column name.

Reply:
@ebyhr thanks!

please make sure the column name is cleanly rejected at some point, so that user knows this is illegal column _name_, not something else.

eg `io.prestosql.plugin.mysql.TestMySqlDistributedQueries#isColumnNameRejected` verifies the actual exception message contains "Incorrect column name"